### ISSUE TITLE
If I choose Bluemix deployment, I still want to know how to run locally

### ISSUE THREAD

**Issue by rvennam** [WASdev/tool.accelerate.core#124]
Currently, if you choose the Bluemix deployment checkbox, it hides the command to run locally (`http://localhost:9080/myLibertyApp`)

Instead, we should show both.

**Comment by katheris** [comment]
Fair point, I wouldn't expect people to just push Bluemix and never want to do anything locally.

**Comment by iainduncani** [comment]
As we also have issue #113 that doesn't actually set Bluemix as a default profile when you run it I wonder if we should just ditch the check box and put both sets of instructions up there all the time and never try to make Bluemix a default profile (similar to the Gradle experience where you always have to explicitly run the cfPush).

**Comment by katheris** [comment]
I've been wondering about not ditching the checkbox, but have it so that if you select Bleumix you get the properties and profile, and if not you don't. So that people can get a simpler pom file if they don't want to deploy to Bluemix. Although if we do that we should probably provide a mechanism to add the Bluemix build code later.

**Comment by iainduncani** [comment]
That's a good call, over a quarter of the POM is dedicated to the Bluemix push at the moment so it would be a good chunk to cut out + simplify

**Comment by rvennam** [comment]
Personally, to deploy to Bluemix I would rather just have a simple manifest file and use cf push. I would not want to put my Bluemix credentials, org and space as maven params as part of every push. 

I say we ditch all the Bluemix additions from pom.xml and simply show cf instructions:
Either: 
`cf push <appname> -p target/JavaHelloWorldApp.war`
or give them a simple manifest.yml which has the `path` filled out and the user just do a `cf push`

Example: https://github.com/IBM-Bluemix/java-helloworld

**Comment by NottyCode** [comment]
If I were doing it manually I would agree, but if I am trying to automate as a part of my devops pipeline wouldn't I want it to be integrated into my maven build?

**Comment by rvennam** [comment]
If we want to cover that case, I would want a checkbox for "Bluemix/CloudFoundry Maven Integration", which to me is clearer than "Bluemix default deployment"
